data/src/main/java/com/example/data/mappers/WeatherDataMapper.kt

```diff
@@ -30,11 +32,19 @@ class WeatherDataMapper {
                     temp = dailyWeatherApi.data[i].max_temp.roundToInt().toString(),
                     icon = "$ICON_URL${dailyWeatherApi.data[i].weather.icon}.png",
                     date = parsingDate(dailyWeatherApi.data[i].datetime),
-                    description = dailyWeatherApi.data[i].weather.description
+                    description = dailyWeatherApi.data[i].weather.description,
```

Comment:
There is also code duplicating. Should be possible to use operator map + method mapWeather.